packages/vite/src/client/overlay.ts

```diff
@@ -177,6 +177,16 @@ export class ErrorOverlay extends HTMLElement {
   constructor(err: ErrorPayload['err'], links = true) {
     super()
     this.root = this.attachShadow({ mode: 'open' })
+
+    if (typeof trustedTypes !== 'undefined') {
+      const sanitizer = trustedTypes.createPolicy('error-overlay', {
+        createHTML: (t: string) => t,
+      })
+      this.root.innerHTML = sanitizer.createHTML(template)
+    } else {
+      this.root.innerHTML = template
+    }
+
```

Comment:
```suggestion

    if (typeof trustedTypes !== 'undefined') {
      const sanitizer = trustedTypes.createPolicy('error-overlay', {
        createHTML: (t: string) => t,
      })
      template = sanitizer.createHTML(template)
    }

```

Is a creating a policy expensive? We we could cache the `sanitizer` out of the constructor so it can be reused later.